src/dcos_e2e/_vendor/dcos_launch/templates/vpc-ebs-only-cluster-template.json

```diff
@@ -256,7 +256,26 @@
                 "ec2:CopySnapshot",
                 "ec2:DeleteSnapshot",
                 "ec2:DescribeSnapshots",
-                "ec2:DescribeSnapshotAttribute"
+                "ec2:DescribeSnapshotAttribute",
```

Comment:
Are we still supporting dcos-launch? Should we remove that?

Reply:
If it still passing tests, we should keep it in. We might drop RHEL support, as this is failing (modified AMI's?). This PR should get us back to passing tests, and then we can discuss other changes.